storage/storage_rules_test.go

```diff
@@ -490,6 +495,110 @@ func TestDBStorageGetContentForRulesRowsError(t *testing.T) {
 	assert.Contains(t, buf.String(), "SQL rows error while retrieving content for rules")
 }
 
+func TestDBStorageToggleRule(t *testing.T) {
+	for _, state := range []storage.RuleToggle{
+		storage.RuleToggleDisable, storage.RuleToggleEnable,
+	} {
+		mockStorage, closer := helpers.MustGetMockStorage(t, true)
+		defer closer()
+
+		mustWriteReport3Rules(t, mockStorage)
+
+		helpers.FailOnError(t, mockStorage.ToggleRuleForCluster(
+			testdata.ClusterName, testdata.Rule1ID, testdata.UserID, state,
+		))
+
+		_, err := mockStorage.GetFromClusterRuleToggle(testdata.ClusterName, testdata.Rule1ID, testdata.UserID)
+		helpers.FailOnError(t, err)
+
+		helpers.FailOnError(t, mockStorage.Close())
```

Comment:
you don't have to call close since closer already does that

Reply:
since it's for loop where you create storage per test I'd wrap it to a function

Reply:
right, I see, thanks :+1: fixd :)